src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -495,6 +495,9 @@ def _build_role_scope(resource_group_name, scope, subscription_id):
         if resource_group_name:
             err = 'Resource group "{}" is redundant because scope is supplied'
             raise CLIError(err.format(resource_group_name))
+        from msrestazure.tools import is_valid_resource_id
```

Comment:
`msrestazure` is not maintained anymore. 

```suggestion
        from azure.mgmt.core.tools import is_valid_resource_id
```

Reply:
There is actually some bug with `is_valid_resource_id`: https://github.com/Azure/azure-sdk-for-python/issues/11658